misc/ls-extensions/modules/performance-analyzer-services/src/main/java/io/ballerina/PerformanceAnalyzerNodeVisitor.java

```diff
@@ -569,6 +581,17 @@ public HashMap<String, Object> getActionInvocations() {
         invocationInfo.put(ACTION_INVOCATION_KEY, this.startNode);
         return invocationInfo;
     }
+    public HashMap<String, Object> getWorkers() {
+
+        this.workers.values().forEach(ParserUtil::getReducedTree);
+        ParserUtil.getReducedTree(this.startNode);
+        this.workers.put("mainWorker", this.startNode);
```

Comment:
Shall we define a constant for mainWorker?

Reply:
Fixed